src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -105,6 +105,34 @@ def stop_process(proc):
 
         return procrtn
 
+    @staticmethod
+    def set_other_env_vars(cartobj):
```

Comment:
we want to make it so that every cart test automatically calls this without having to do anything extra